### ISSUE TITLE
Variant Annotation Subworkflow

### ISSUE THREAD

**Issue by nicola-calonaci** [caravagnalab/nextflow_modules#21]
The subworkflow is not working properly.

I submitted the usual SLURM job

```
#!/bin/sh
#### Cluster specific arguments ####

#SBATCH --job-name=NF_MASTER
#SBATCH --output=logs/nf_%j.out
#SBATCH --error=err
#SBATCH --partition=EPYC
#SBATCH --nodes=1
#SBATCH --ntasks=23
#SBATCH --cpus-per-task=1
#SBATCH --mem=64G
#SBATCH --time=12:00:00
#SBATCH -A cdslab

module load singularity/3.10.4
module load java/1.8.0
module load R/4.2.3

RESULTS_PATH=$evoverse/results/

/orfeo/LTS/CDSLab/LT_storage/ncalonaci/nextflow/nextflow run $evoverse/main.nf \
    -c $evoverse/modules/config/nextflow_config_slurm \
    --samples $evoverse/sample_input.csv \
    --publish_dir $RESULTS_PATH \
    --tools vep,annovar
```

to use both VEP and ANNOVAR, but the `results` directory only contains VEP files:

```
results/
└── CLL
    ├── CT339
    │   ├── 37
    │   │   └── VEP
    │   │       └── vep.vcf.gz
    │   ├── 38
    │   │   └── VEP
    │   │       └── vep.vcf.gz
    │   ├── 39
    │   │   └── VEP
    │   │       └── vep.vcf.gz
    │   └── 40
    │       └── VEP
    │           └── vep.vcf.gz
``` 

First, I do not have any output from  the `vcf2maf` process, where I woul expect maf files to be produced.

Moreover, the pipeline fails with the following error message 
```
Error executing process > 'VARIANT_ANNOTATION:ANNOVAR_ANNOTATE (9)'

Caused by:
  Process `VARIANT_ANNOTATION:ANNOVAR_ANNOTATE (9)` terminated with an error exit status (2)

Command executed:

  mkdir -p CLL/CT344/41/ANNOVAR
  gzip -dc 61-vs-41.somaticSNV.vcf.gz > vcf_File



  perl /orfeo/LTS/CDSLab/LT_storage/nextflow_modules/annovar//table_annovar.pl \
  vcf_File \
  /orfeo/LTS/CDSLab/LT_storage/nextflow_modules/ANNOVAR/humandb_38/ \
  -buildver hg38 \
  -out CLL/CT344/41/ANNOVAR/annovar \
  -protocol refGene \
  -operation g \
  -nastring . \
  -vcfinput \
  -polish \
  -remove \
  -thread 4

  awk '{print (NR>1?"CT344/41":"Tumor_Sample_Barcode") "        " $0}' CLL/CT344/41/ANNOVAR/annovar.hg38_multianno.txt > CLL/CT344/41/ANNOVAR/annovar.txt

Command exit status:
  2

Command output:
  (empty)

Command error:
  INFO:    Converting SIF file to temporary sandbox...
  Can't open perl script "/orfeo/LTS/CDSLab/LT_storage/nextflow_modules/annovar//table_annovar.pl": No such file or directory
  INFO:    Cleaning up image...
  ERROR:   failed to delete container image tempDir /orfeo/LTS/CDSLab/LT_storage/ncalonaci/tmp/rootfs-748387108: unlinkat /orfeo/LTS/CDSLab/LT_storage/ncalonaci/tmp/rootfs-748387108/root: directory not empty
```

**Comment by nicola-calonaci** [comment]
Solved with commit a8499feda6d9ffcbe57c68e9a3c5322b349f4f7d

**Comment by nicola-calonaci** [comment]
Solved with commit a8499feda6d9ffcbe57c68e9a3c5322b349f4f7d
